Formula/c/cyphernetes.rb

```diff
@@ -0,0 +1,29 @@
+class Cyphernetes < Formula
+  desc "Kubernetes Query Language"
+  homepage "https://cyphernet.es"
+  url "https://github.com/AvitalTamir/cyphernetes/archive/refs/tags/v0.11.0.tar.gz"
+  sha256 "2bb8945493178c7173b45dbeeac4c668d49d9010cab265185324d88567429549"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    ENV["GOBIN"] = buildpath/"bin"
+    ENV.prepend_path "PATH", buildpath/"bin"
+    system "go", "install", "modernc.org/goyacc@v1.0.3"
```

Comment:
Why is this needed? Doesn't it just define this in the dependency file?

Reply:
It's not a normal runtime dependency. `goyacc` is invoked from the shell at build time to generate a parser. As far as I know, I need the above to have it available on the path. I could be wrong however, I'm new to Go.

Reply:
@SMillerDev -- Upstream confirmed that this is intended. Please let me know if it's unacceptable, and we'll work on a resolution.

Reply:
I am adding goyacc so that we can directly have it as build dep rather than having a "go install" step